docs/display-settings.md

```diff
@@ -3,16 +3,25 @@
 ## Menu Items
 
 - **Units:**
-Select the units (meters or feet) for the application to display
+Select the units (Metric [meters] or Imperial [feets]) for the application to display
 
 - **Theme:**
-Select the application menu theme (light or dark)
+Select the application menu theme (Light or Dark)
+
+- **Enable Always-On-Top mode:**
+The application will be over all windows.
+
+- **Opacity:**
+Select the application opacity.
 
 - **Enable Replay Menu:**
 Enable the [Replay Menu](replay-data.md)
 
+- **Enable Real-Time Replay:**
+Each profile message will be displayed as the real sensor and not periodically.
```

Comment:
> Each profile message will be displayed as the real sensor and not periodically.

-> Shows playback as it was recorded (ie what you would see from a screen recording when the data was captured). Sets profile messages to display based on recorded timestamps, instead of using a constant period/update rate.